kubernetes/cache/cache.go

```diff
@@ -352,17 +351,9 @@ func (in *kialiCacheImpl) GetZtunnelPods(cluster string) []v1.Pod {
 		return ztunnelPods
 	}
 
-	dsPods, err := kubeCache.GetPods(daemonsets[0].Namespace, "")
+	ztunnelPods, err = kubeCache.GetPods(daemonsets[0].Namespace, fmt.Sprintf("app=%s", config.Ztunnel))
```

Comment:
Can you add a test for `GetZtunnelPods` where the pod names do not have the `config.Ztunnel` prefix to catch this in the future?

Reply:
Done! 

But notice that now the pod names are not used anymore, just the app ztunnel label selector. Should we update this?